packages/app/src/NodeDetailsSidePanel.tsx

```diff
@@ -260,6 +274,72 @@ export default function NodeDetailsSidePanel({
     ],
   });
 
+  const { data: logsTableMetadata } = useTableMetadata(tcFromSource(logSource));
```

Comment:
Not a fan of duplicating things across pod/node/namespace details side panel. We will need to merge them